lib/validation/validators.js

```diff
@@ -85,21 +85,27 @@ module.exports = Validators = {
   },
 
   // eslint-disable-next-line no-inline-comments, spaced-comment
-  before(value/*, [diff, unit][, diff, unit]*/) {
-    const args = [].slice.call(arguments, 1);
-    let date = moment(value, dateFormat);
-    let diff;
-    let unit;
-    while (args.length) {
-      diff = args.shift();
-      unit = args.shift() || 'years';
-      date = date.add(diff, unit);
+  before(value, date/*, [diff, unit][, diff, unit]*/) {
```

Comment:
Also, I initially started thinking by looking at the comment it would be like this before('2017-01-01', [18, 'years'], [11, 'months]) but I then worked out it's before('2017-01-01', [18, 'years', 11, 'months'])
The question, is it still little misleading? It's very minor point, I was thinking of [diff, unit.., diff, unit]